documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
```

Comment:
The first sentence should tell the readers what TWAP is, before going into the details. e.g. "The TWAP strategy is a common algorithmic execution strategy used for splitting up large orders over time."

The second sentence is almost there. Maybe change it to "The TWAP strategy code demonstrates a few useful tools you can use to create sophisticated behaviors in writing tradings strategies:"